### DISCUSSION TITLE
With the remove extentions with DNS with bookmarklets disabled

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#448] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do i get it to work im on version 109.0.5414.94

**Comment by obkduishinijf** [comment, marked as answer]
copy the script and run it in the omnibox (search bar) instead

**Reply by Aka-but-nice** [reply]
How on earth will that work? That will not work in a million years

**Reply by Aka-but-nice** [reply]
Now, I haven't tested this method yet, but try to make the bookmarklet a search engine, if you don't understand what I mean, ask or a video.

**Reply by ghost** [reply]
wym bookmarklet a search engine

**Reply by obkduishinijf** [reply]
> How on earth will that work? That will not work in a million years

since the javascript is able to be ran through bookmarks (just a quick way of putting it into the top search bar,) you can do it by simply typing the javascript into the search bar and running it that way, unless javascript is disabled.

**Reply by Aka-but-nice** [reply]
Ik about that, I use it all the time, because I have to many bookmarklets, the question was, that wouldn't work if the users **policies** blocked javascript:// urls

**Reply by piplup7575** [reply]
This would not work.

**Reply by obkduishinijf** [reply]
> Ik about that, I use it all the time, because I have to many bookmarklets, the question was, that wouldn't work if the users **policies** blocked javascript:// urls

they never said if it was blocked by policy, they said bookmarks

**Reply by Aka-but-nice** [reply]
> > Ik about that, I use it all the time, because I have to many bookmarklets, the question was, that wouldn't work if the users **policies** blocked javascript:// urls
> 
> they never said if it was blocked by policy, they said bookmarks

The title of this thread includes the words "with bookmarklets disabled" there is no way to get bookmarklets if they're blocked, this will not be possible.

**Reply by obkduishinijf** [reply]
> > > Ik about that, I use it all the time, because I have to many bookmarklets, the question was, that wouldn't work if the users **policies** blocked javascript:// urls
> > 
> > 
> > they never said if it was blocked by policy, they said bookmarks
> 
> The title of this thread includes the words "with bookmarklets disabled" there is no way to get bookmarklets if they're blocked, this will not be possible.

yes, that's why i said to run the javascript INSIDE the omnibox (top search bar) instead of doing bookmarklet, since bookmarklets just auto-fill it with a specific link that you specify, so it will still work

**Reply by Aka-but-nice** [reply]
This wouldn't work, it is a "javascript://" URL, specifically in the block list, by admins, if you're entering it into the search bar, you're doing exactly what they blocked.
